packages/editor-ui/src/components/ValueSurvey.vue

```diff
@@ -145,16 +212,17 @@ export default mixins(workflowHelpers).extend({
 				height: 36px;
 			}
 
-			button {
+			.button {
 				margin-left: 10px;
-				font-size: var(--font-size-s);
+
+				button {
+					font-size: var(--font-size-s);
+				}
```

Comment:
why is this needed? we should not have to dig deep to override a component settings? is there a prop we can change here to update this to match? otherwise I would push back against the designs

Reply:
@mutdmour It is needed because of the design and product requirement. I was searching for prop in N8N-button component to change the font-size, but there is no prop for it.